src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related
+ * CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const rulesTrimAndSplit = sourceString
+    .split(';')
+    .map(elem => elem.trim().split(':'));
+
+  rulesTrimAndSplit.forEach(rule => {
+    if (rule[0] && rule[1]) {
+      styles[rule[0].trim()] = rule[1].trim();
+    }
+  });
```

Comment:
5. [CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead.
ForEach is actually a loop, so think of a different way of solving it. My recommendation is to practice reduce method in this task.